katalog/nginx/README.md

```diff
@@ -24,7 +24,28 @@ Fury distribution Ingress NGINX is deployed with the following configuration:
 
 ## Deployment
 
-You can deploy NGINX by running the following command in the root of the project:
+Our Ingress module by default is deployed as a `DaemonSet`, meaning that it will try to deploy 1 ingress-controller pod on every worker node.
+
+This is probably NOT what you want, standard Fury clusters have at least 1 `infra` node (nodes that are dedicated to run Fury infrastructural components, like prometheus, elasticsearch, and the ingress controllers).
+
+If your cluster has `infra` nodes you should patch the daemonset adding the `NodeSelector` for the `infra` nodes to the Ingress `DaemonSet`. You can do this usiing the following kustomize patch:
+
+```yaml
+---
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: nginx-ingress-internal
```

Comment:
This is the single nginx, there is no distinction between internal and external

Reply:
Thank you! I've updated the patches examples